srt-protocol/src/protocol/sender/buffer.rs

```diff
@@ -472,7 +470,7 @@ mod test {
             recv_tsbpd_latency: TSBPD,
             cipher: None,
             stream_id: None,
-            bandwidth: LiveBandwidthMode::default(),
+            bandwidth: Default::default(),
```

Comment:
Seems like this change is a loss of visual information, but not huge deal

Reply:
I started using the pattern elsewhere, so I just did this for consistency. I'm mostly indifferent, with a slight bias now for Default::default(). Inference, in general hides all kinds of visual information, more isn't always better.... :)